mmv1/templates/terraform/examples/cloudbuildv2_connection.tf.erb

```diff
@@ -0,0 +1,16 @@
+provider "google-beta" {}
+
+resource "google_project" "project" {
```

Comment:
Do we need to use a new project here, or can we manage these resources in an existing one?

Reply:
Removed project.